.idea/dataSources.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
-    <data-source source="LOCAL" name="ehrenamtskarte@localhost" uuid="d83188d7-bad5-4b46-b4d7-13cabcdaf7c7">
+    <data-source source="LOCAL" name="ehrenamtskarte@db-postgis" uuid="d83188d7-bad5-4b46-b4d7-13cabcdaf7c7">
```

Comment:
Please undo this.
If you want to avoid this, maybe try this:
1) open database view in intellij
2) rightclick on "ehrenamtskarte@db-postgis" and properties
3) adjust the name in the first line

Reply:
Nice, thank you! That works :)